src/utils/isOnline.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Assumes a browser is online if `undefined`, otherwise makes a best effort
+ * @link https://developer.mozilla.org/en-US/docs/Web/API/NavigatorOnLine/onLine
+ */
+export function isOnline() {
+  return navigator.onLine === undefined || navigator.onLine;
```

Comment:
Is `navigator` available in a Node environment?

Reply:
Great catch 🎉

Reply:
Updated and added tests to check for the expected behaviors.